Add Python to CompositeImplicitAutograd

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #82648
* #82647

Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

ezyang committed:
Add Python to CompositeImplicitAutograd

Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

zou3519 reviewed (approved):
lint seems legit; also there are source control markers in the expect files

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]

ezyang committed:
Update on "Add Python to CompositeImplicitAutograd"


Signed-off-by: Edward Z. Yang <ezyangfb.com>

Differential Revision: [D38354502](https://our.internmc.facebook.com/intern/diff/D38354502/)

[ghstack-poisoned]
